docs/governance/proposals/index.md

```diff
@@ -11,5 +11,17 @@ Each proposal type has different Governance Parameters:
 | Snapshot                     | https://snapshot.org/#/hbot-prp.eth | https://snapshot.org/#/hbot-ip.eth | https://snapshot.org/#/hbot.eth |
 | Vote Duration                | 7 days                    | 14 days                 | 14 days                  |
 | Min HBOT Balance             | 1                         | 10,000                  | 50,000                   |
-| Quorum Percentage            | 0.1% of HBOT total supply | 3% of HBOT total supply | 10% of HBOT total supply |
+| Quorum Percentage            | 0.1% of HBOT circulating supply | 3% of HBOT circulating supply | 10% of HBOT circulating supply |
 | Approval Threshold           | >50% of tokens approved   | >50% of tokens approved | >50% of tokens approved  |
+
+## Minimum Quorum Updates
+
+After the Approval of [HGP-3](https://snapshot.org/#/hbot.eth/proposal/0x9a58b2136b5c20af9d4fd507cd8174e7f6e21425549bbe864e089154f76da415) the minimum Quorum needed for the approval of proposal is now a percentage of the circulating supply.
+
+*Last Update: 02/15/2022*
+
+|                       |                |
+|-----------------------|----------------|
+|**Total Supply**       | 1,000,000,000  |
+|**Circulating Supply** | 554,483,484    |
```

Comment:
I don't think this is correct. This should equal the outstanding circulating supply, which equals the total tokens claimed, as well as those distributed to the community via governance.

Update this monthly. As of Jan 31, 2022, circulating supply is 27,169,236 HBOT tokens, the total amount claimed.

Reply:
The reference i used is the amount that is still on the Foundation treasury (https://gnosis-safe.io/app/eth:0x6caB2E6bd7E3F95792a61bC37eF24D11C79cD166/balances)

Should we not consider the vesting contracts? Altough they are locked, they can still be used to vote through the `delegate_call` function.

Reply:
That's not what circulating supply means. Circulation supply is the amount of tokens in circulation.